### PR TITLE
Sign up user for municipality after creating package

### PR DESCRIPTION
**Pull request by Valioesi**
Resolves: https://expedition-grundeinkommen.monday.com/boards/853311751/pulses/1832417693

After package was created successfully the user is signed up for the municipality if not already the case. I did not introduce the state of the rest call, it's just being executed in the background. 

The user is not being signed up for the newsletter of the municipality though. I was unsure if it was the right place to do it. I can add that though. 

To test: go to mensch/{userId}/paket-nehmen?campaignCode={a campaign you are not signed up for, e.g. berlin-1) and create a package. You should be signed up for berlin then. If you replicate the steps, no new api call should be executed.

### CODE REVIEW

**Review by nmauersberg — APPROVED** [review]
Great!

**Comment by Valioesi** [comment]
I added updating newsletter settings after creating a package. 
One is signed up for the newsletter of this campaign, if
- user has newsletterConsent true and
- user does not already have newsletter of campaign (or has not already unsubscribed from it)

File: src/components/PledgePackage/CreatePledgePackage/index.js [hunk 1]
```diff
@@ -45,6 +48,49 @@ export default ({ userData, updateCustomUserData }) => {
...
+            name: mapCampaignCodeToState(campaignCode),
+            ags,
+            value: true,
+            extraInfo: false,
+            timestamp: new Date().toISOString(),
+          },
+        ];
+      }
+
+      console.log('about to update user with data', data);
```
Comment (nmauersberg) [inline src/components/PledgePackage/CreatePledgePackage/index.js:88]: Should this log be removed?

**Review by nmauersberg — APPROVED** [review]
Looks good, thank you!

File: src/components/PledgePackage/CreatePledgePackage/index.js (hunk 1 above)
Comment (Valioesi) [reply src/components/PledgePackage/CreatePledgePackage/index.js:88]: Yup, thanks, my bad!

**Pull request merged**